.qubesbuilder

```diff
@@ -0,0 +1,15 @@
+host:
+  rpm:
+    build:
+    - libvirt.spec
+    - libvirt-python.spec
+source:
+  files:
+  - url: https://libvirt.org/sources/libvirt-6.6.0.tar.xz
+    signature: https://libvirt.org/sources/libvirt-6.6.0.tar.xz.asc
+    pubkeys: &id001
+    - redhat-jiri-denemark-key.asc
+    - redhat-veillard-key.asc
+  - url: https://libvirt.org/sources/python/libvirt-python-6.6.0.tar.gz
+    signature: https://libvirt.org/sources/python/libvirt-python-6.6.0.tar.gz.asc
+    pubkeys: *id001
```

Comment:
Better avoid those yaml tricks here (and possibly ban them in the qubes-builder parser).